roles/core/log_client/readme.rst

```diff
@@ -34,13 +34,62 @@ Log client verbosity defaults to info, it can be one of the following (defined i
 | debug    | debug-level messages             |
 +----------+----------------------------------+
 
+You can force usage of a static file content instead of a generated one by 
+setting **log_client_static_configuration** into inventory.
+**log_client_static_configuration** is a multi lines variable that should contain
+the full desired content of the target rsyslog.conf file.
+
+For example:
+
+.. code-block:: yaml
+
+  log_client_static_configuration: |
+    # rsyslog configuration file
+
+    # For more information see /usr/share/doc/rsyslog-*/rsyslog_conf.html
+    # If you experience problems, see http://www.rsyslog.com/doc/troubleshoot.html
+
+    #### MODULES ####
+
+    # The imjournal module bellow is now used as a message source instead of imuxsock.
+    $ModLoad imuxsock # provides support for local system logging (e.g. via logger command)
+    $ModLoad imjournal # provides access to the systemd journal
+    #$ModLoad imklog # reads kernel messages (the same are read from journald)
+    #$ModLoad immark  # provides --MARK-- message capability
+    ...
+
+| You can also split the configuration into several files, using **log_client_custom_conf**. 
+| This variable takes your custom files name as a list (without .conf).
+| Each configuration is specified under log_client_**name**_conf, as is **log_client_static_configuration**.
+|
+| For example:
+
+.. code-block:: yaml
+
+  log_client_custom_conf:
+    - local_rules
+    - forwarding_rules
+
+  log_client_local_rules_rules: |
```

Comment:
Might be **log_client_local_rules_conf**? (ok for the _conf here since the file will be local_rules.conf)